OpenSim/Examples/ExampleMain/OutputReference/TugOfWar_Complete.cpp

```diff
@@ -115,7 +115,7 @@ int main()
         
         Cylinder cylGeometry(0.2, .3);
         cylGeometry.setFrameName("CylAnchor");
-        cylGeometry.setRepresentation(Geometry::DrawWireframe);
+        cylGeometry.setRepresentation(DecorativeGeometry::DrawWireframe);
```

Comment:
Is this the same as `VisualRepresenation` below? Why does Geometry expose it? Isn't it an attribute of the Appearance?

If you have multiple Geometry share the same Appearance, what does it mean for the Geometry to set an attribute of the shared appearance? Does it get its own copy at that point or do the appearance of all the objects that share that Appearance suddenly change?